dapp/src/services/Wallet.js

```diff
@@ -1149,22 +1149,31 @@
       */
       wallet.confirmTransaction = function (address, txId, options, cb) {
         var instance = Web3Service.web3.eth.contract(wallet.json.multiSigDailyLimit.abi).at(address);
-        instance.confirmTransaction.estimateGas(txId, wallet.txDefaults(), function (e, gas){
-          if (e) {
-            cb(e);
-          }
-          else {
-            Web3Service.sendTransaction(
-              instance.confirmTransaction,
-              [
-                txId,
-                wallet.txDefaults({gas: Math.ceil(gas * 1.5)})
-              ],
-              options,
-              cb
-            );
-          }
-        });
+        Web3Service.sendTransaction(
+          instance.confirmTransaction,
+          [
+            txId,
+            wallet.txDefaults({gas: 300000})
```

Comment:
Why is this value hard coded here?

Reply:
There are a couple of reasons. If you pass no value, in the UI a much higher value than needed will appear and there is a restriction that you cannot go under that value (that's using the UI, later in Metamask you could tweak it) so it was a bit inconvenient. The value passed is hardcoded because what was failing was the estimation for the gas parameter, and I chose 300000 because on Kovan (where the gas estimation was working due to the base fee being the minimum) most of the estimations were in the range of 150000 and 200000, so 300000 seemed reasonable and was already being used as a hardcoded value in other parts of the code.